code/modules/atmos_automation/implementation/vent_pump.dm

```diff
@@ -88,4 +88,202 @@
 					injector_names|=I.id_tag
 			vent_pump = input("Select a vent:", "Vent Pumps", vent_pump) as null|anything in injector_names
 			parent.updateUsrDialog()
-			return 1
\ No newline at end of file
+			return 1
+
+/datum/automation/set_vent_pump_pressure//controls the internal/external pressure bounds of a vent pump.
+	name = "Vent pump: Pressure settings"
+
+	var/vent_pump = null
+	var/intpressureout = 0//these 2 are for DP vents, if it's a unary vent you're sending to it will take intpressureout as var
+	var/intpressurein = 0
+	var/extpressure = 0
+	var/mode = 0//0 for unary vents, 1 for DP vents.
+
+	Export()
```

Comment:
>The pathing in all the other automation defines was relative too, I just followed it like it was

@PJB3005 What we do when we encounter relative pathing, color macro usage, etc in a file we're working in is to make it conform to current standards of span classes and absolute paths.